coreos-4/policies/SI-System_and_Information_Integrity/component.yaml

```diff
@@ -7,54 +7,1196 @@
+  narrative:
+    - text: |
+        A control response for SI-7(12) is planned. Engineering progress can be
+        tracked via:
+
+        https://issues.redhat.com/browse/CMP-555
+
+- control_key: SI-10 (3)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - text: |
+        A control response for SI-7(12) is planned. Engineering progress can be
+        tracked via:
+
+        https://issues.redhat.com/browse/CMP-556
+
+- control_key: SI-10 (4)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - text: |
+        A control response for SI-7(12) is planned. Engineering progress can be
+        tracked via:
+
+        https://issues.redhat.com/browse/CMP-557
+
+- control_key: SI-10 (5)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: ''
+    - text: |
+        This control reflects organizational procedures/policies that
+        are not applicable to the configuration of CoreOS.
+
+- control_key: SI-11
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: planned
+  narrative:
+    - key: a
+      text: |
+        The customer will be responsible for generating error messages in
+        cusomer-controlled operating systems and software that provide
+        information necessary for corrective actions without revealing
+        information that could be exploited by adversaries. A successful
+        control response will need to discuss the process by which error
+        messages are created, analyzed, and corrected when necessary.
+
+        A control response for SI-11(a) is planned. Engineering progress can be
+        tracked via:
+
+        https://issues.redhat.com/browse/CMP-392
+    - key: b
+      text: |
+        The customer will be responsible for revealing error messages only to
+        authorized personnel. A successful control response will need to
+        address how error messages are displayed and the means by which access
+        to error messages is controlled.
+
+        A control response for SI-11(b) is planned. Engineering progress can be
+        tracked via:
+
+        https://issues.redhat.com/browse/CMP-293
 
 - control_key: SI-12
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: planned
   narrative:
     - text: |
-        'The customer will be responsible for handling and retaining
+        The customer will be responsible for handling and retaining
         information within, or hosted by, CoreOS in
         accordance with applicable federal laws, Executive Orders,
         directives, policies, regulations, standards, and operational
         requirements. A successful control response will need to outline
         the specific requirements applicable to customer information
         handling and retention, and the means by which those requirements
-        are met.'
```

Comment:
Thanks for removing those damned ticks. It's my fault they're in there... earlier tools needed them. Needed to be updated for some time!